src/demo/native/story.js

```diff
@@ -59,21 +121,52 @@ storiesOf('Demos', module)
           width: 40px;
           height: 40px;
         }
+        .input-field {
```

Comment:
Why not "input-text"?

Reply:
Do you mean the name of the class, or why I'm not using AXAInput?

Reply:
axainput

Reply:
it would be a hard coupling to a holistic component

Reply:
Why? This is a normal story, like all the other stories are.

Reply:
Yes, but there are no stories where other  axa components are being used.

Reply:
I agree with @NoahLiechti. In stories we dont have lerna support and coupling hardly via import `axa-input-text` is not good. Is only for demo purposes so its ok as it is IMHO